tianshou/utils/logger/wandb.py

```diff
@@ -29,16 +32,94 @@ class WandBLogger(BaseLogger):
     :param int test_interval: the log interval in log_test_data(). Default to 1.
     :param int update_interval: the log interval in log_update_data().
         Default to 1000.
+    :param str project: TODO. Default to "tianshou".
+    :param str name: TODO. Default to None.
+    :param str entity: TODO. Default to None.
+    :param str run_id: TODO. Default to None.
```

Comment:
@AyushExel could you please fill out this part?

Reply:
@Trinkle23897 Done